test/parallel/test-net-connect-local-error.js

```diff
@@ -6,6 +6,7 @@ const net = require('net');
 const server = net.createServer();
 server.listen(0);
 const port = server.address().port;
+console.log(`server port ${port}`);
```

Comment:
Ack.
I just added it to try to check something.
That test is interacting with the test of this PR.